core/src/test/scala/org/apache/spark/deploy/security/KafkaTokenUtilSuite.scala

```diff
@@ -78,6 +79,21 @@ class KafkaTokenUtilSuite extends SparkFunSuite with BeforeAndAfterEach {
     Configuration.setConfiguration(null)
   }
 
+  test("checkProxyUser with proxy current user should throw exception") {
+    val currentUser = UserGroupInformation.getCurrentUser()
```

Comment:
Use `UGI.createUserForTesting` instead of messing with the current user. (Or even `createProxyUser` might just work.)

Reply:
Fixed.